part_1/title_IX/sekcja_56_PL.md

```diff
@@ -0,0 +1,35 @@
+
+(1) Funkcjonariusz policji ma prawo zatrzymać: broń kategorii A, B, C lub D, amunicję, myśliwski czarny proch, proch bezdymny, zapałki, pozwolenie, licencję na broń, świadectwo broni, listy przewozowe dotyczące stałego wywozu, przywozu lub tranzytu broni i amunicji, zezwolenie na wywóz produktów, które są objęte kontrolą handlu w Czechach ze względów bezpieczeństwa, Europejską Kartę Broni, pozwolenie na transport lub zgłoszenie transportu broni i amunicji z artykułu 50 lub 50a; aby zapobiec takim działaniom ich posiadaczy, które powodują nie spełnienie obowiązków lub łamamie zakazów ustanowionych ustawą.
+<!-- dosł: aby zapobiec ich właścicielom takich działań, które naruszają obowiązek lub nie przestrzegają zakazu ustanowionego w ustawie, ale to lepiej brzmi. Potrzebne znalezienie, czy dobrze przedłumaczyłam szczególnie 'černý lovecký prach, bezdýmný prach, zápalky', bo wydaje się to... dziwne. -->
+
+(2) Funkcjonariusz policji ma prawo zatrzymać: broń kategorii A, B lub C, amunicję do tej broni, pozwolenie, licencję na broń, świadectwo broni, listy przewozowe dotyczące stałego wywozu, przywozu lub tranzytu broni i amunicji, zezwolenie na wywóz produktów, które są objęte kontrolą handlu w Czechach ze względów bezpieczeństwa, Europejską Kartę Broni, w której są zapisane bronie z kategorii A, B lub C, pozwolenie na transport lub zgłoszenie transportu broni i amunicji z artykułu 50 lub 50a, jeśli
+
+a) istnieje uzasadnione podejrzenie, że u posiadacza pozwolenia na broń doszło do takiej zmiany stanu zdrowia, która może w związku z obsługą broni stanowić zagrożenie dla jego życia lub zdrowia albo życia lub zdrowia innych osób,
+
+b) cofnięto : pozwolenie na broń zgodnie z artykułem 27 ust. 1 punkt a), b) lub c); lub licencję na broń zgodnie z artykułem 36 ust. 1 punkt b) lub artykułem 36 ust. 2; a uprzedni posiadacz karty broni palnej lub pozwolenia na broń nie oddał świadectwa broni, broni, amunicji i zakazanych akcesoriów do broni właściwemu wydziałowi policji lub   
+<!-- potrzebne sprawdzenie w poprzednich artykułach: różnica między 'zbrojní průkaz' i  'průkaz zbraně', to pierwsze jest b) - ja przetłumaczyłam jako pozwolenie na broń -->
+
+c) istnieją podstawy do zatrzymania zgodnie z artykułem 57 ust. 1 i istnieje uzasadnione podejrzenie, że dalsza obsługa broni może stanowić zagrożenie dla jego życia lub zdrowia albo życia lub zdrowia innych osób.
+
+(3) Jeżeli posiadacz pozwolenia na broń lub posiadacz licencji na broń na wezwanie funkcjonariusza policji natychmiast nie zrzeknie się przedmiotów, które zostały zatrzymane na podstawie ust. 2, lub nie ujawni miejsca ich przechowywania i uniemożliwi policji ich przejęcie, funkcjonariusz policji ma prawo: wejść do lokalu mieszkalnego, innych pomieszczeń lub terenu, jeśli można rozsądnie założyć, że znajdują się tam przedmioty _(dop. tłumacza:, o których mowa)_ ; i zatrzymać je.
+<!-- teren - szukam odpowiednika, coś jak grunt, ziemia - posiadana przez kogoś-->
```

Comment:
Teren to je teren. W propozycji nowelizacji art. 25 KK autorstwa Z. Ziobry była mowa o ogrodzonym terenie. W czeskim tekście nic nie świadczy o tym, że musi być akurat ogrodzony, więc zostawiłbym teren.